packages/webpack/src/config/base.js

```diff
@@ -118,7 +122,9 @@ export default class WebpackBaseConfig {
     const env = {
       'process.env.NODE_ENV': JSON.stringify(this.mode),
       'process.mode': JSON.stringify(this.mode),
-      'process.static': this.buildContext.isStatic
+      'process.dev': this.dev,
+      'process.static': this.target === 'static',
```

Comment:
I guess it'd make sense to have a utility object or function that covers all possible targets :thinking:

Reply:
I thought about it too, how would you implement it?

Reply:
If we don't have sth. like `utils/constants.js`, I'd introduce it and then use a named export for the targets.

Reply:
Will update thanks!
The targets are `server`, `serverless` and `static` :)